extras/systemd/dynflow-sidekiq@.service

```diff
@@ -1,16 +1,26 @@
 [Unit]
-Description=Foreman jobs daemon on sidekiq
+Description=Foreman jobs daemon - %i on sidekiq
 Documentation=https://theforeman.org
 After=network.target remote-fs.target nss-lookup.target
 
 [Service]
-Type=forking
+Type=simple
 User=foreman
 TimeoutSec=90
 Environment=RAILS_ENV=production
 Environment=DYNFLOW_SIDEKIQ_SCRIPT=/usr/share/foreman/extras/dynflow-sidekiq.rb
+# Greatly reduce Ruby memory fragmentation and heap usage
+# https://www.mikeperham.com/2018/04/25/taming-rails-memory-bloat/
+Environment=MALLOC_ARENA_MAX=2
```

Comment:
At a cost of reducing CPU performance, it's worth noting. Probably not a big deal for dynlflow but it's always good to see some numbers when doing fine tuning like this.

Reply:
For the regular dynflow this has been tested extensively by the Satellite performance team and they found it leads to a much more stable installation.